KristianJBorgwarth/Golferino
# Lms/feat/playerround update

added update of playerround totalscore 

How to test?
* Go to DBMS --> Change a score
* open swagger and run the playerround/udate on the playerround of the score you just updated
* Confirm the playerround.totalscore has been updated
